buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -973,7 +974,7 @@ private void logFileContents(String description, Path from, boolean tailLogs) {
                         String previousMessage = normalizeLogLine(ring.getLast());
                         if (MESSAGES_WE_DONT_CARE_ABOUT.stream().noneMatch(previousMessage::contains)
                             && (previousMessage.contains("ERROR") || previousMessage.contains("WARN"))) {
-                            errorsAndWarnings.put(previousMessage, errorsAndWarnings.getOrDefault(previousMessage, 0) + 1);
+                            errorsAndWarnings.put(ring.getLast(), errorsAndWarnings.getOrDefault(previousMessage, 0) + 1);
```

Comment:
Can you elaborate on this change is for?

Reply:
Yes, `previousMessage` is the "normalized" form of the original line which has the timestamp removed. Logging this versions was awkward, as well as missed the context of the timestamp or the original message. This change simply prints the original full message instead of the truncated version.